### PR TITLE
refactor: custom constants

### PR DESCRIPTION
**Pull request by johnletey**
😅

### CODE REVIEW

**Review by t8 — APPROVED** [review]
LGTM. Amazing job. Looks like @littledivy will be able to commence unit tests once we get this merged!

File: src/index.ts [hunk 1]
```diff
@@ -33,7 +33,15 @@ export default class Verto {
...
 
-  constructor(keyfile?: JWKInterface, arweave?: Arweave) {
+  public exchangeContract!: string;
+  public exchangeWallet!: string;
+
+  constructor(
+    keyfile?: JWKInterface,
+    arweave?: Arweave,
+    exchangeContract?: string,
+    exchangeWallet?: string
```
Comment (littledivy) [inline src/index.ts]: Why not have an interface called Options and have these fields inside it? It would be more clean :)

**Review by littledivy — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (t8) [reply src/index.ts]: Brilliant idea. I second.

File: src/utils/constants.ts [hunk 2]
```diff
@@ -1,7 +1,3 @@
-export const exchangeContractSrc =
-  "usjm4PCxUd5mtaon7zc97-dt-3qf67yPyqgzLnLqk5A";
-export const exchangeWallet = "aLemOhg9OGovn-0o4cOCbueiHT9VgdYnpJpq7NgMA1A";
-
```
Comment (littledivy) [inline src/utils/constants.ts]: Could've still been in `constants.ts` and used as default values when argument is not provided.

File: src/index.ts [hunk 3]
```diff
@@ -43,10 +51,21 @@ export default class Verto {
...
+    if (exchangeContract) {
+      this.exchangeContract = exchangeContract;
+    } else {
+      this.exchangeContract = "usjm4PCxUd5mtaon7zc97-dt-3qf67yPyqgzLnLqk5A";
+    }
+    if (exchangeWallet) {
+      this.exchangeWallet = exchangeWallet;
+    } else {
+      this.exchangeWallet = "aLemOhg9OGovn-0o4cOCbueiHT9VgdYnpJpq7NgMA1A";
+    }
```
Comment (littledivy) [inline src/index.ts]: Ditto. Corresponding to prev comment

**Review by littledivy — CHANGES_REQUESTED** [review]
Yay! Some minor changes...

File: src/index.ts (hunk 3 above)
Comment (littledivy) [reply src/index.ts]: Done

File: src/utils/constants.ts (hunk 2 above)
Comment (littledivy) [reply src/utils/constants.ts]: Done

File: src/index.ts (hunk 1 above)
Comment (littledivy) [reply src/index.ts]: Done

**Review by littledivy — APPROVED** [review]
LGTM! Very helpful for automating test

**Pull request merged**